src/utilities.jl

```diff
@@ -71,3 +71,7 @@ end
         return acos(x)
     end
 end
+
+
+# some place holders for package level function names
+function origin end
```

Comment:
documentation string would be useful otherwise few people will understand what this function definition is for. Maybe show an example of overloading this function?